sdk/purview/purview-catalog-rest/package.json

```diff
@@ -84,7 +84,7 @@
   "autoPublish": false,
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
-    "@azure-rest/core-client": "1.0.0-beta.5",
+    "@azure-rest/core-client": "1.0.0-beta.6",
```

Comment:
can you update the changelog for this package as well?

Reply:
Version for purview package is not updated in this PR. what's the change log you are asking for?